src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  border-radius: 4px;
+}
+
+.search-bar,
+.search-bar__label,
+.search-bar__input {
+  width: 100%;
+}
```

Comment:
But why? You set other styles for all these classes, it's better to avoid splitting styles to different selectors, just add this rule to all these selectors styles